manager/manager/templatetags/manager.py

```diff
@@ -232,3 +232,13 @@ def has_expired(errors) -> bool:
 
 
 register.filter("has_expired", has_expired)
+
+@register.simple_tag
+def url_replace(request, field, value):
```

Comment:
There's no docstring to explain what this function does. AFAIU it returns the (encoded) GET query params with the updated value for the requested field